src/main/java/ch/njol/skript/util/LiteralUtils.java

```diff
@@ -40,12 +40,19 @@ public class LiteralUtils {
 	 * @param <T>  {@code expr}'s type
 	 * @return {@code expr} without {@link UnparsedLiteral} objects
 	 */
-	@SuppressWarnings("unchecked")
+	@SuppressWarnings({"unchecked"})
```

Comment:
This change isn't needed
```suggestion
	@SuppressWarnings("unchecked")
```